src/snowflake/cli/api/utils/rendering.py

```diff
@@ -93,6 +95,21 @@ def get_sql_cli_jinja_env():
     )
 
 
+def get_template_cli_jinja_env(template_root: SecurePath) -> Environment:
```

Comment:
I would suggest moving this into `plugins.init` module to avoid too many templatings in one place. WDYT?

Reply:
I had a different thought - to have all template definitions is one place :D
IMO it would be convenient for maintenance - we know which '<* ... *>' go with which context.

If we want to move this function closer to its plugin, I'd suggest doing the same with the rest of them

Reply:
Nice idea - this way they still would be close and easy to find/differentiate, but we won't mix context in a single file

Reply:
Refactor rendering module: https://github.com/snowflakedb/snowflake-cli/pull/1270/commits/18c86b2af3b367038e7b2516f96bc698e46bf562

Refactor `snow init` code: https://github.com/snowflakedb/snowflake-cli/pull/1270/commits/448ebd8069d22af33f9be158229ca5eafb763142